go/kbfs/libkbfs/leveldb.go

```diff
@@ -98,6 +99,15 @@ func (ldb *LevelDb) PutWithMeter(key, value []byte, putMeter *CountMeter) (
 	return ldb.Put(key, value, nil)
 }
 
+// StatStrings returns newline-split leveldb stats, suitable for JSONification.
+func (ldb *LevelDb) StatStrings() ([]string, error) {
+	stats, err := ldb.GetProperty("leveldb.stats")
+	if err != nil {
+		return nil, err
+	}
+	return strings.Split(stats, "\n"), nil
+}
```

Comment:
Curiosity question: any reason to split this when we are joining them with `\n` again when printing out?

Reply:
It's for when they are printed in a JSON format, so that each individual line in the string array can be printed in an aligned way.  See any log send for how the existing cache stats look.

Reply:
@songgao it's so they're a `[]string` for the log send json status, was an easy way to get it to print nicely

Reply:
ah that makes sense. Thanks